foreword.qmd

```diff
@@ -3,7 +3,7 @@
 I've always felt that something was off with existing approaches
 to geospatial data science in other programming languages.
 I remember sitting in the beautiful
-[Stanford's Green library](https://en.wikipedia.org/wiki/Cecil_H._Green_Library)
+[Cecil H. Green library at Stanford](https://en.wikipedia.org/wiki/Cecil_H._Green_Library)
```

Comment:
Please preserve the informal name of the library. This is how we used to call it. The formal name can be found in the Wikipedia article.

Reply:
Sure, I think my main objective here was to move the Stanford. You usually would place an adjective before noun in focus which is Green Library. So I think either you should do "I remember sitting in Stanford's beautiful [Green Library]" or "I remember sitting in the beautiful [Green Library](...) at Stanford"